reinforcement_learning/rlclientlib/console_tracer.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include "trace_logger.h"
+namespace reinforcement_learning {
```

Comment:
nit: newline after includes and after namespace

Reply:
done.